### PR TITLE
Try improved status embed formatting

### PR DESCRIPTION
**Pull request by wqyeo**
Based on #47 

Currently, [this](https://i.imgur.com/Q6dG5r8.png) is the new style.<Br>(Posted it on to the discord server too, by the way.)

Made each respective section of the status on to each embed field.

The footer now contains the up-time, it shows in this format:<br>`x Years; x Months; x Days; x Hours; x Minutes;`
It will not show any of the respective time if the value is `0` for that time.

(P.S There is a few lines of repeated function calls with different param inputs; Please give some suggestion on how to improve on that if possible 😅 )

### CODE REVIEW

File: Cake.Core/Discord/Embeds/CakeModuleEmbeds.cs [hunk 1]
```diff
@@ -11,17 +12,101 @@ public class CakeModuleEmbeds
...
+                        .WithValue($"{Main.GetClient().Latency}ms")
+                    );
+
+                statusFields.Add(
+                    new EmbedFieldBuilder().WithName("Shards")
+                       .WithValue($"{Main.GetClient().Shards.Count}")
+                    );
+
+                statusFields.Add(
+                    new EmbedFieldBuilder().WithName("Guild Shard")
```
Comment (CakeAndBanana) [inline Cake.Core/Discord/Embeds/CakeModuleEmbeds.cs]: Combine Guild shard and Shards to 1 field, Like this: "(guildShard)/(totalShards)". No need to seperate them.

**Review by CakeAndBanana — CHANGES_REQUESTED** [review]
Maybe not use semi columns for the spacing between the time, use , or ○ between it.
The guild shards and total shards can be combined.

**Comment by wqyeo** [comment]
@CakeAndBanana 👍 
This is now the new format after the update: https://i.imgur.com/NokjsZg.png

It seperates the times by `,` rather than semi-colons.

**Comment by CakeAndBanana** [comment]
I think this looks fine enough, I will push this to master :).
Thanks for the PR!

About the repeated function calls, I couldnt really think of a solution. Maybe later I will find one but I will tell you on Discord.

**Pull request merged**